members/templates/members/department_list.html

```diff
@@ -37,7 +37,7 @@
     <ul class="nav nav-pills nav-fill">
     {% for region, depList in departments.items %}
     <li  class="nav-item">
-        {% if region == 'Sjælland' %}
+        {% if region == 'Hovedstaden' %}
```

Comment:
Du skal tilføje en ekstra fane, vi har nu 5 regioner hvor vi før havde 4.

Reply:
![2020-02-11_12h44_40](https://user-images.githubusercontent.com/17178122/74234051-4998a000-4ccc-11ea-9db2-36fa818eafb7.png)
 No need, for loopet på linie 52 laver jo bare en for hver region i systemet, så det ser allerede ud som det skal

Reply:
Det er da rigtigt! 
Det er mig der roder rundt i det, det er endda mig der har skrevet den løkke der gør det.